dnf-behave-tests/features/obsoletes.feature

```diff
@@ -9,14 +9,23 @@ Background: Use dnf-ci-obsoletes repository
   Given I use repository "dnf-ci-obsoletes"
 
 
-Scenario: Install of obsoleted package, but higher version than obsoleted present
+Scenario: Install of obsoleted package, but higher version than obsoleted present, both paths are valid picked based on lexicographical order
    When I execute dnf with args "install PackageA"
    Then the exit code is 0
     And Transaction is following
         | Action        | Package                                   |
         | install       | PackageA-0:3.0-1.x86_64                   |
 
 
+@bz1902279
+Scenario: Install of obsoleted package, but higher version than obsoleted present, both paths are valid picked based on lexicographical order
```

Comment:
The two scenarios now have the same name, can you differentiate them? (to keep the scenario name shorter, I'd leave the details of how it works out of it and put them in a comment)

Reply:
Definitely. :slightly_smiling_face: